parkings/models/parking.py

```diff
@@ -40,14 +51,14 @@ def __str__(self):
         start = localtime(self.time_start).replace(tzinfo=None)
 
         if self.time_end is None:
-            return "%s -> (%s)" % (start, self.registration_number)
+            return "%s -> (%s)" % (start, 'ABC-123')
 
         if self.time_start.date() == self.time_end.date():
             end = localtime(self.time_end).time().replace(tzinfo=None)
         else:
             end = localtime(self.time_end).replace(tzinfo=None)
 
-        return "%s -> %s (%s)" % (start, end, self.registration_number)
+        return "%s -> %s (%s)" % (start, end, 'ABC-123')
```

Comment:
TODO comment might help remembering to remove these after demoing needs are gone.